contribute/breaking-changes-guide.md

```diff
@@ -0,0 +1,120 @@
+# Handling breaking changes in our frontend API
+
+This guide tries to help you identifying and communicating breaking changes introduced to our frontend API.
+
+- [What are our public APIs?](#what-are-our-public-apis)
+- [What is Levitate?](#what-is-levitate)
+- [How does the CI workflow look like?](#how-does-the-ci-workflow-look-like)
+- [I received a comment on my PR, what does it mean?](#i-received-a-comment-on-my-pr-what-does-it-mean)
+- [**I know it's a breaking change, what's next?**](#i-know-its-a-breaking-change-whats-next)
+  - [Introduce breaking changes only in major versions](#introduce-breaking-changes-only-in-major-versions)
+  - [Deprecate first](#deprecate-first)
+  - [Communicate](#communicate)
+- [I still have questions, who can help me out?](#i-still-have-questions-who-can-help-me-out)
+
+---
+
+## What are our public APIs?
+
+The Grafana frontend codebase is exposing functionality through NPM packages to make plugin development easier and faster.
+These packages live in the `/packages` folder and contain packages like:
+
+- `@grafana/data`
+- `@grafana/runtime`
+- `@grafana/toolkit`
+- `@grafana/ui`
+- etc. ([View all available on npmjs.com](https://www.npmjs.com/org/grafana))
```

Comment:
This link highlights a lot of packages that levitate currently doesn't consider. Do we want to limit this link to only cover repo?

```suggestion
- etc. ([They can be viewed here](https://github.com/grafana/grafana/tree/main/packages)
```

Reply:
Aaa you are right. 👍